test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesTestHelpers.cs

```diff
@@ -28,7 +28,19 @@ public static async Task InvokeAndValidateHttpTrigger(EndToEndTestFixture fixtur
             string masterKey = await fixture.Host.GetMasterKeyAsync();
             uri = $"api/{functionName}?code={masterKey}&name=Mathew";
             request = new HttpRequestMessage(HttpMethod.Get, uri);
+            request.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue("text/plain"));
+            response = await fixture.Host.HttpClient.SendAsync(request);
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            Assert.Equal("text/plain", response.Content.Headers.ContentType.MediaType);
+            Assert.Equal("Hello Mathew", body);
```

Comment:
This test was missing assertions.

Reply:
You're right - this helper intended to make the same request with the master key but never sent the request! Let's just fix this helper by sending the request and doing the same validations as above, and do your content type tests in a separate test. This helper is used in many places to send/validate requests, so we shouldn't be adding a bunch of combination testing in here.

Reply:
That's fair! The reason I added here is that the callers for this function are the E2E test anytime they want to test Http scenario. To me it made sense that any time we test Http, we should also cover the absence of content types.
But I see your point to separate this out and intentionally add tests for content types that only targets the scenario.

Ok, I will make that update. Your suggestion makes more sense design wise.